ui/perfherder/constants.js

```diff
@@ -1,2 +1,8 @@
 export const tValueCareMin = 3; // Anything below this is "low" in confidence
 export const tValueConfidence = 5; // Anything above this is "high" in confidence
+
+// Backend server endpoints
+export const endpoints = {
+    issueTrackers: '/performance/issue-tracker/',
```

Comment:
Could you switch this to 2 space indent please (about to be enforced by the linter in #4276 for everything that's not in the legacy `ui/js/` directory)